turbopack/crates/turbopack-core/src/chunk/chunk_group.rs

```diff
@@ -26,40 +27,50 @@ pub async fn make_chunk_group(
     chunk_group_entries: impl IntoIterator<Item = ResolvedVc<Box<dyn Module>>>,
     availability_info: AvailabilityInfo,
 ) -> Result<MakeChunkGroupResult> {
+    let can_split_async = !matches!(
+        *chunking_context.environment().chunk_loading().await?,
+        ChunkLoading::Edge
+    );
+
     let ChunkContentResult {
-        chunk_items,
-        external_output_assets,
-        external_module_references,
+        chunkable_modules,
         async_modules,
         traced_modules,
+        passthrough_modules,
         forward_edges_inherit_async,
         local_back_edges_inherit_async,
         available_async_modules_back_edges_inherit_async,
-    } = chunk_content(chunking_context, chunk_group_entries, availability_info).await?;
+    } = chunk_content(
+        chunk_group_entries,
+        availability_info,
+        can_split_async,
+        *chunking_context.is_tracing_enabled().await?,
```

Comment:
could you move that into a variable similar to `can_split_async` for consistency